django_fastdev/apps.py

```diff
@@ -458,6 +459,11 @@ def collect_valid_blocks(template, context):
                     # 'isinstance(x, (AutoEscapeControlNode, BlockNode, FilterNode, ForNode, IfNode,
                     # IfChangedNode, SpacelessNode))' at the risk of missing some we don't know about
                     result |= collect_nested_blocks(x)
+                else:
+                    if settings.FASTDEV_INVALID_BLOCKS_CALLBACK is not None:
```

Comment:
This check will crash if the setting does not exist. I assume you want to use getattr() here.